example/astx/util/litconv.go

```diff
@@ -25,22 +23,17 @@ func RuneValue(lit []byte) rune {
 	return r
 }
 
-/*
-Convert the literal value of a scanned token to int64
-*/
+// UintValue will attempt to parse a byte-slice as a signed base-10 64-bit integer.
```

Comment:
But this comment looks much better and conforms to the go standard in an important place and I really like it.